va/va.h

```diff
@@ -597,12 +597,16 @@ typedef struct _VAConfigAttrib {
 
 /** @name Attribute values for VAConfigAttribEncSliceStructure */
 /**@{*/
-/** \brief Driver supports an arbitrary number of rows per slice. */
-#define VA_ENC_SLICE_STRUCTURE_ARBITRARY_ROWS           0x00000000
 /** \brief Driver supports a power-of-two number of rows per slice. */
 #define VA_ENC_SLICE_STRUCTURE_POWER_OF_TWO_ROWS        0x00000001
-/** \brief Driver supports an arbitrary number of rows per slice. */
+/** \brief Driver supports an arbitrary number of MBs per slice. */
 #define VA_ENC_SLICE_STRUCTURE_ARBITRARY_MACROBLOCKS    0x00000002
+/** \brief Dirver support 1 rows per slice */
+#define VA_ENC_SLICE_STRUCTURE_EQUAL_ROWS               0x00000004
+/** \brief Dirver support max encoded slice size */
+#define VA_ENC_SLICE_STRUCTURE_MAX_SLICE_SIZE           0x00000008
+/** \brief Driver supports an arbitrary number of rows per slice. */
+#define VA_ENC_SLICE_STRUCTURE_ARBITRARY_ROWS           0x00000010
```

Comment:
Can all flags be OR'ed together? If not, could you please document it clearly?  How about to define VA_ENC_SLICE_STRUCTURE_ARBITRARY_ROWS to 0x08 because VA_ENC_SLICE_STRUCTURE_POWER_OF_TWO_ROWS / VA_ENC_SLICE_STRUCTURE_ARBITRARY_MACROBLOCKS / VA_ENC_SLICE_STRUCTURE_EQUAL_ROWS / VA_ENC_SLICE_STRUCTURE_ARBITRARY_ROWS define slice structure per slice.